docs/loop-3/features.md

```diff
@@ -121,38 +121,38 @@ This section is a continuation of the information presented in the [Accept Recom
+When the `Bolus` screen is entered directly from the toolbar, the button choices are `Enter Bolus` if none is recommended, `Deliver` if a value is on the `Bolus` row, or `Cancel` using the button on the upper left. The user can also tap on the value on the `Bolus` row to bring up a keyboard to modify that amount. When doing that, the value is automatically set to zero.
 
-There are other alert messages that might be displayed if the pump or CGM is not active.  Those are found on the [Loop 3 Displays](displays-v3.md) page.
+Other alert messages might be displayed if the pump or CGM is not active.  Those are found on the [Loop 3 Displays](displays-v3.md) page.
 
 The two graphics below are examples of manual bolus screens. 
 
 * In the first graphic, no bolus was recommended
     * If you tap on the `Enter Bolus` button at the bottom, it brings up a screen to enable you to type in an amount and then `Deliver` it. 
     * Alternatively, you can tap the `0` amount in the `Bolus` row and perform the same action as the `Enter Bolus` button
-    * If you do not want to override the recommendation, hit the `Cancel` button at upper left.
+    * If you do not want to override the recommendation, hit the `Cancel` button at the upper left.
 
 ![manual bolus when no bolus recommended](img/loop-3-manual-bolus-no-rec.svg){width="350"}
 {align="center"}
 
 * In the second graphic, a recommended amount is shown
     * If you tap on `Deliver` that recommended dose is delivered
     * If you tap on the value on the `Bolus` row, you can override the amount
-    * The amount displayed on the `Bolus` row is modified to 0 U with the first tap - at that point, you may enter a new value or tap `Cancel` using the button at upper left of the screen
+    * The amount displayed on the `Bolus` row is modified to 0 U with the first tap - at that point, you may enter a new value or tap `Cancel` using the button at the upper left of the screen
 
 ![manual bolus when bolus is recommended](img/loop-3-manual-bolus-rec.svg){width="350"}
 {align="center"}
 
 
 ## Remote Carb / Bolus
 
-Loop 3 has a Remote Carb and Remote Bolus feature to enable remote caregivers to better assist the person who needs support managing with Loop.
+Loop 3 includes a feature for remote input of Carbohydrates and Bolus, enabling caregivers who are not physically present to better assist the individual who requires support in managing Loop.
 
-The author of this feature wrote this [Remote Carb/Bolus Guide](https://docs.google.com/document/d/1wPpCljo9NuwllltjhImf7YZReIgqP9yF05PN7E6hphM){: target="_blank" } documentation. Please read carefully and use with the appropriate level of caution.
+The author of this feature wrote this [remote carb/bolus guide](../nightscout/remote-overview.md) documentation. Please read carefully and use with the appropriate level of caution.
```

Comment:
ℹ️ 💡 My understanding is that _Google Translate_ (GT) considers capitalized words as not to be translated, and it keeps them verbatim. 
If I use the same words in lowercase, the translation will be correct.

Reply:
Actually - that guide was removed after I incorporated the contents into the Nightscout section but it got spread over the 5 pages. Maybe replace with:

Please read the documentation about this feature carefully to ensure proper configuration and use:
* [Remote Command Overview](../nightscout/remote-overview.md)

Reply:
Done.
commit [362e753](https://github.com/LoopKit/loopdocs/pull/786/commits/362e753a0ab86c7c362d90ce888b12e160137a86) fixes the first sentence and the label of links.